ccb_vaultic/Makefile

```diff
@@ -0,0 +1,67 @@
+# Pick up cross-compiler info from environment
```

Comment:
Not working for me....

```% make
I. -O2 -DHAVE_CCBVAULTIC -DCCBVAULTIC_DEBUG -DCCBVAULTIC_DEBUG_TIMING -DCCBVAULTIC_NO_SHA -I../../wolfssl -DTARGETCHIP_VAULTIC_420 -DUSE_SPI -IVaultIC-TLS_420/vaultic_tls-4xx/VaultIC-Elib_420/src/common -IVaultIC-TLS_420/vaultic_tls-4xx/VaultIC-Elib_420/src/device/vaultic_4XX_family  -IVaultIC-TLS_420/vaultic_tls-4xx/lib/VaultIC_420_TLS_Lib/target/raspberry/dist -DWC_USE_DEVID=0x56490420 -DBENCH_EMBEDDED -DNO_MAIN_DRIVER   -c -o ccb_vaultic.o ccb_vaultic.c
make: I.: No such file or directory
make: [ccb_vaultic.o] Error 1 (ignored)
I. -O2 -DHAVE_CCBVAULTIC -DCCBVAULTIC_DEBUG -DCCBVAULTIC_DEBUG_TIMING -DCCBVAULTIC_NO_SHA -I../../wolfssl -DTARGETCHIP_VAULTIC_420 -DUSE_SPI -IVaultIC-TLS_420/vaultic_tls-4xx/VaultIC-Elib_420/src/common -IVaultIC-TLS_420/vaultic_tls-4xx/VaultIC-Elib_420/src/device/vaultic_4XX_family  -IVaultIC-TLS_420/vaultic_tls-4xx/lib/VaultIC_420_TLS_Lib/target/raspberry/dist -DWC_USE_DEVID=0x56490420 -DBENCH_EMBEDDED -DNO_MAIN_DRIVER   -c -o ../../wolfssl/wolfcrypt/test/test.o ../../wolfssl/wolfcrypt/test/test.c
make: I.: No such file or directory
make: [../../wolfssl/wolfcrypt/test/test.o] Error 1 (ignored)
I. -O2 -DHAVE_CCBVAULTIC -DCCBVAULTIC_DEBUG -DCCBVAULTIC_DEBUG_TIMING -DCCBVAULTIC_NO_SHA -I../../wolfssl -DTARGETCHIP_VAULTIC_420 -DUSE_SPI -IVaultIC-TLS_420/vaultic_tls-4xx/VaultIC-Elib_420/src/common -IVaultIC-TLS_420/vaultic_tls-4xx/VaultIC-Elib_420/src/device/vaultic_4XX_family  -IVaultIC-TLS_420/vaultic_tls-4xx/lib/VaultIC_420_TLS_Lib/target/raspberry/dist -DWC_USE_DEVID=0x56490420 -DBENCH_EMBEDDED -DNO_MAIN_DRIVER   -c -o main-test.o main-test.c
make: I.: No such file or directory
make: [main-test.o] Error 1 (ignored)
o wolfcrypt-test ccb_vaultic.o ../../wolfssl/wolfcrypt/test/test.o main-test.o -L../../wolfssl/src/.libs -lwolfssl -lm -LVaultIC-TLS_420/vaultic_tls-4xx/lib/VaultIC_420_TLS_Lib/target/raspberry/dist/VaultIC_420_TLS_Lib-SPI-Release -lvaultic420_tls
make: o: No such file or directory
make: [wolfcrypt-test] Error 1 (ignored)
I. -O2 -DHAVE_CCBVAULTIC -DCCBVAULTIC_DEBUG -DCCBVAULTIC_DEBUG_TIMING -DCCBVAULTIC_NO_SHA -I../../wolfssl -DTARGETCHIP_VAULTIC_420 -DUSE_SPI -IVaultIC-TLS_420/vaultic_tls-4xx/VaultIC-Elib_420/src/common -IVaultIC-TLS_420/vaultic_tls-4xx/VaultIC-Elib_420/src/device/vaultic_4XX_family  -IVaultIC-TLS_420/vaultic_tls-4xx/lib/VaultIC_420_TLS_Lib/target/raspberry/dist -DWC_USE_DEVID=0x56490420 -DBENCH_EMBEDDED -DNO_MAIN_DRIVER   -c -o ../../wolfssl/wolfcrypt/benchmark/benchmark.o ../../wolfssl/wolfcrypt/benchmark/benchmark.c
make: I.: No such file or directory
make: [../../wolfssl/wolfcrypt/benchmark/benchmark.o] Error 1 (ignored)
I. -O2 -DHAVE_CCBVAULTIC -DCCBVAULTIC_DEBUG -DCCBVAULTIC_DEBUG_TIMING -DCCBVAULTIC_NO_SHA -I../../wolfssl -DTARGETCHIP_VAULTIC_420 -DUSE_SPI -IVaultIC-TLS_420/vaultic_tls-4xx/VaultIC-Elib_420/src/common -IVaultIC-TLS_420/vaultic_tls-4xx/VaultIC-Elib_420/src/device/vaultic_4XX_family  -IVaultIC-TLS_420/vaultic_tls-4xx/lib/VaultIC_420_TLS_Lib/target/raspberry/dist -DWC_USE_DEVID=0x56490420 -DBENCH_EMBEDDED -DNO_MAIN_DRIVER   -c -o main-bench.o main-bench.c
make: I.: No such file or directory
make: [main-bench.o] Error 1 (ignored)
o wolfcrypt-benchmark ccb_vaultic.o ../../wolfssl/wolfcrypt/benchmark/benchmark.o main-bench.o -L../../wolfssl/src/.libs -lwolfssl -lm -LVaultIC-TLS_420/vaultic_tls-4xx/lib/VaultIC_420_TLS_Lib/target/raspberry/dist/VaultIC_420_TLS_Lib-SPI-Release -lvaultic420_tls
make: o: No such file or directory
make: [wolfcrypt-benchmark] Error 1 (ignored)
davidgarske@Davids-MBP-2 ccb_vaultic % ls -la VaultIC-TLS_420/vaultic_tls-4xx/VaultIC-Elib_420/src/common
total 288
drwxrwxr-x@ 16 davidgarske  staff    512 Apr 26  2022 .
drwxrwxr-x@  6 davidgarske  staff    192 Apr 26  2022 ..
drwxrwxr-x@ 11 davidgarske  staff    352 Apr 26  2022 comms
-rw-rw-r--@  1 davidgarske  staff   4955 Apr 26  2022 vaultic_apdu.h
-rw-rw-r--@  1 davidgarske  staff  12747 Apr 26  2022 vaultic_command.c
-rw-rw-r--@  1 davidgarske  staff   5495 Apr 26  2022 vaultic_command.h
-rw-rw-r--@  1 davidgarske  staff   3959 Apr 26  2022 vaultic_common.h
-rw-rw-r--@  1 davidgarske  staff   7233 Apr 26  2022 vaultic_crc16.c
-rw-rw-r--@  1 davidgarske  staff   4959 Apr 26  2022 vaultic_crc16.h
-rw-rw-r--@  1 davidgarske  staff  47785 Apr 26  2022 vaultic_err.h
-rw-rw-r--@  1 davidgarske  staff   5558 Apr 26  2022 vaultic_mem.h
-rw-rw-r--@  1 davidgarske  staff   5845 Apr 26  2022 vaultic_options.h
-rw-rw-r--@  1 davidgarske  staff   3987 Apr 26  2022 vaultic_timer_delay.h
-rw-rw-r--@  1 davidgarske  staff   5580 Apr 26  2022 vaultic_typedefs.h
-rw-rw-r--@  1 davidgarske  staff   6818 Apr 26  2022 vaultic_utils.c
-rw-rw-r--@  1 davidgarske  staff   5317 Apr 26  2022 vaultic_utils.h
davidgarske@Davids-MBP-2 ccb_vaultic %```

Reply:
This is expecting to have a bunch of environment defines (like: export NDK_CC=/path/to/ndk/cc) to use the NDK cross-compiler.  I put some notes in the readme on how to set this up.  Lemme send you the finalized zip file for you to dump into your directory.  I had to make a few changes to the startup sequence and the makefile to also use the same NDK compiler.